index.html

```diff
@@ -6,60 +6,62 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
     <title>Assignment 1</title>
+    <script src="login.js"></script>
 </head>
 
 <body>
     <main>
         <div class="container">
             <div class="container__left">
```

Comment:
Can we extra div wrapper here. Ie. use just 1 div instead of 2 (container__left & info-area) if possible.

Reply:
ok sir